helloworld-rs/README-source.adoc

```diff
@@ -0,0 +1,57 @@
+include::../shared-doc/attributes.adoc[]
+
+= helloworld-rs: Jakarta RESTful Web Services Helloworld Example
+:author: Ashwin Mehendale
+:level: Beginner
+:technologies: Jakarta RESTful Web Services
+:openshift: true
+:portedToGlow: false
```

Comment:
in case you don't overwrite this with the helloworld README, please remember to move this to true

Reply:
done